client/my-sites/google-my-business/select-business-type/index.js

```diff
@@ -106,7 +138,7 @@ class GoogleMyBusinessSelectBusinessType extends Component {
 					target="_blank"
 					onClick={ this.trackCreateYourListingClick }
 				>
-					{ translate( 'Create Your Listing', {
+					{ translate( 'Create Listing', {
```

Comment:
Hi! I've found a possible matching string that has already been translated **18** times:
`translate( 'Create My Listing' )` *ES Score: 12*
See 1 additional suggestion in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/update/gmb-flows)

Help me improve these suggestions: react with :thumbsdown: if the suggestion doesn't make any sense, or with :thumbsup: if it's a particularly good one (even if not implemented).